llm_studio/src/utils/utils.py

```diff
@@ -31,7 +31,10 @@ def set_seed(seed: int = 1234) -> None:
 
 def set_environment(cfg):
     """Sets and checks environment settings"""
-
+    if not cfg.environment.openai_api_token:
+        cfg.environment.openai_api_token = os.getenv("OPENAI_API_KEY", "")
+    if not cfg.logging.neptune_api_token:
+        cfg.logging.neptune_api_token = os.getenv("NEPTUNE_API_TOKEN", "")
     os.environ["OPENAI_API_KEY"] = cfg.environment.openai_api_token
```

Comment:
i guess, this line is not needed as we are using the key with the line below. But doesn't need to be addressed here.